ccd-definition/CaseEventToFields/actionCMO.json

```diff
@@ -273,7 +307,20 @@
     "DisplayContext": "MANDATORY",
     "PageID": "nextHearing",
     "PageLabel": " ",
-    "PageDisplayOrder": 10,
+    "PageDisplayOrder": 11,
     "PageColumnNumber": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "actionCMO",
+    "CaseFieldID": "cmoToAction",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "NOT_SHOWN",
+    "PageLabel": " ",
+    "PageDisplayOrder": 12,
+    "PageColumnNumber": 1,
+    "PageShowCondition": "actionCMOPlaceholderHeading=\"DO_NOT_SHOW\""
```

Comment:
Do we need it here? maybe for readability we could add this on page 1, if it's needed to use in subsequent conditions?

Reply:
If I had this one page one I’d need FieldShowCondition which would remove the reference to in subsequent pages no?

Reply:
changed this to page one and tested, seems to be working